packages/jmix-react-core/src/timer/timer.ts

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect, useCallback } from "react";
+
+interface TimerHookControls {
+  start: () => void, 
+  stop: () => void
+};
+
+type TimerHook = (
+  delay: number,
+  timerCalbback: () => void,
```

Comment:
typo `timerCalbback`

Reply:
also, as we are already in context of timer I suggest to use `callback instead od timerCallback`